packages/services/api/src/modules/activity/resolvers/MemberLeftActivity.ts

```diff
@@ -0,0 +1,11 @@
+import type { ActivityObject } from '../../../shared/entities';
+import type { MemberLeftActivityResolvers } from './../../../__generated__/types.next';
+
+export const MemberLeftActivity: MemberLeftActivityResolvers = {
+  __isTypeOf(activity) {
+    return activity.type === 'MEMBER_LEFT';
+  },
+  email(activity: any) {
```

Comment:
Do you think we could as part of this PR make this more type-safe?

Reply:
Let me see if I can wrangle this type in reasonable time 👀

Reply:
I've looked into this and we could fix the `any` type issue now, but there's a few awkward things we need to do. 

Here's what I've tried:
1. Add `ActivityObject` as one of the implementing type mappers e.g.
```ts
export type ProjectCreatedActivityMapper = ActivityObject;
```
2. Run generate
3. Static analysis detects fields to update:
<img width="961" alt="Screenshot 2024-07-09 at 8 26 23 PM" src="https://github.com/kamilkisiela/graphql-hive/assets/33769523/e2228c4a-5c87-4090-af77-8669c497a860">
Here's how we can fix these issues:
- For (a.), update codegen config scalar `DateTime` to be `Date | string` as this is the correct expected type for `DateTime`
- For (b.) and (c.), make `ProjectCreatedActivityMapper`'s `project` and `user` fields to be required i.e. have a separate type from `ActivityObject`. We would need to make a mapper for every Activity type.

Since this change both codegen config and is quite verbose, I've marked it as a cleanup task to do at the end. This helps us finish the migration quickly, and then we can make code improvements after